micah-akpan/questioner-api
# #163344537 Refactor validation error status codes

#### What does this PR do?
Uses appropriate validation error status codes for API requests data
#### Description of Task to be completed?
Refactor code to use the appropriate API validation error status codes
#### How should this be manually tested?
- clone repo
- `npm install`
- `npm run dev:start`
- Launch Postman and access http://localhost:9999/api/v1/auth/signup with firstname, lastname, email and password to sign up and receive an auth token
- Add the received token to `Authorization` Header 
- Access http://localhost:9999/api/v1/auth/login, intentionally omit the email field or use a wrong email format, say bob, instead of bob@email.com
You should receive a 400 error, instead of a previous 422
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163344537
#### Screenshots (if appropriate)
#### Questions:
